device/device.go

```diff
@@ -0,0 +1,15 @@
+package device
+
+import "errors"
+
+var errDataLenInvalid = errors.New("invalid data size on port")
+
+// IODevice describes the interface a IO-Port device must implement regardless of the
+// bus it is attached to.
+// Clean up and unifying pci.Device and IODevice of this package will be required.
```

Comment:
Yeah, I think so too. We don't have to deal with it in this PR, but I'd like to work on it eventually.